xknx/knxip/tpdu.py

```diff
@@ -0,0 +1,104 @@
+"""
+Created on 14.01.2021.
+
+@author: bonzius
+"""
+from xknx.telegram import IndividualAddress, GroupAddress, Telegram, TPDUType
```

Comment:
```suggestion
from __future__ import annotations

from xknx.telegram import IndividualAddress, GroupAddress, Telegram, TPDUType
```
so you can use `list[int]` instead of `typing.List[int]` 
and `GroupAddress | IndividualAddress | InternalGroupAddress` instead of `Union[...]`

Your return type annotations didn't work because
- `List` is imported when `if TYPE_CHECKING` is True - so only on mypy runs.
- `-> TPDU` would need to be a string literal `-> "TPDU"` because no postponed evaluation of annotations without __future__.annotations. With the import you can use it directly.